pkg/restore/restore.go

```diff
@@ -864,16 +865,20 @@ func (ctx *context) restoreItem(obj *unstructured.Unstructured, groupResource sc
 			}
 
 			if shouldRenamePV {
-				// give obj a new name, and record the mapping between the old and new names
-				oldName := obj.GetName()
-				newName, err := ctx.pvRenamer(oldName)
-				if err != nil {
-					addToResult(&errs, namespace, errors.Wrapf(err, "error renaming PV"))
-					return warnings, errs
+				if oldName == obj.GetName() {
+					// pvRestorer hasn't modified the PV name, we need to rename the PV
+					name, err = ctx.pvRenamer(oldName)
```

Comment:
I haven't fully traced it, but I see that we're now updating the `name` variable which is already declared above this scope -  is there any potential for that to cause an issue by updating in-place? I'll try to trace it through some more and see.

Reply:
Yes, We are using `name` during logging and it may create some confusion the user about source PV. I have renamed it to `pvName`.